screens/NewsFeed.jsx

```diff
@@ -4,107 +4,109 @@ import { layout } from '../GlobalStyles';
 import NewsCard from '../components/NewsCard';
 import NewsHeader from '../components/NewsHeader';
 import Error from '../components/Error';
-import fetchPredictions from '../utils/AxiosRequest';
+import { fetchPredictions, fetchAllArticles } from '../utils/AxiosRequest';
 import ERRORACTIONS from '../constants/ErrorActions';
-import SplashScreen from '../components/SplashScreen';
+import SplashScreen from '../components/SplashScreen'; 
 
 export default function NewsFeedScreen() {
-	const [recommendedArticles, setRecommendedArticles] = useState([]);
-	const [loading, setLoading] = useState(true);
-	const [isRefreshing, setIsRefreshing] = useState(false);
-	const userID = "1812344"; 
+  const [subview, setSubview] = useState(1);
+  const [articles, setArticles] = useState([]);
+  const [loading, setLoading] = useState(true);
+  const [isRefreshing, setIsRefreshing] = useState(false);
+  const userID = "1812344"; 
 
-	const handleRefresh = async () => {
-		setIsRefreshing(true);
-		await fetchArticles(false);
-		setIsRefreshing(false);
-	}
+  const fetchData = async (loadMore) => {
+    try {
+      let data;
+      switch (subview) {
+        case 1:
+          data = await fetchPredictions(userID, 10);
+          break;
+        case 3:
```

Comment:
forstår ikke helt hvad case 1 og 3 er. Er det muligt at være mere specifik / skrive en kommentar hvad tallene står for? Vi kunne evt bruge enum?